MaterialDesignThemes.Wpf.Tests/ColorDefinitionsTests.cs

```diff
@@ -0,0 +1,102 @@
+using System.Windows;
+using System.Windows.Media;
+using Xunit;
+
+namespace MaterialDesignThemes.Wpf.Tests
+{
+    public class ColorDefinitionsTests
+    {
+        public ColorDefinitionsTests()
+        {
+            if (!UriParser.IsKnownScheme("pack"))
+                Assert.NotNull(new Application());
+        }
+
+        [Fact]
+        public void VerifyLightXamlColorsInTheme()
+        {
+            AssertXamlColorsInTheme("MaterialDesignTheme.Light.xaml", new MaterialDesignLightTheme());
+        }
+
+        [Fact]
+        public void VerifyDarkXamlColorsInTheme()
+        {
+            AssertXamlColorsInTheme("MaterialDesignTheme.Dark.xaml", new MaterialDesignDarkTheme());
+        }
+
+        private static void AssertXamlColorsInTheme(string xaml, IBaseTheme baseTheme)
+        {
+            var theme = new Theme();
+            theme.SetBaseTheme(baseTheme);
+
+            var uri = new Uri("pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/" + xaml);
+            var resourceDictionary = new ResourceDictionary { Source = uri };
+
+            foreach (var (key, solidColorBrush) in resourceDictionary
+                .OfType<DictionaryEntry>()
+                .Where(e => e.Value is SolidColorBrush)
+                .Select(e => ((string) e.Key, (SolidColorBrush) e.Value))
+                .OrderBy(e => e.Item1))
+            {
+                var baseThemePropertyName = key == "ValidationErrorBrush"
+                    ? "ValidationErrorColor"
+                    : key;
+                var baseThemeProperty = baseTheme.GetType().GetProperty(baseThemePropertyName);
+                Assert.False(baseThemeProperty == null, $"{baseThemePropertyName} from {xaml} not found in {baseTheme.GetType()}");
+
+                var themePropertyName = key == "ValidationErrorBrush"
+                    ? "ValidationError"
+                    : key.Replace("MaterialDesign", "");
+                var themeProperty = theme.GetType().GetProperty(themePropertyName);
+                Assert.False(themeProperty == null, $"{themePropertyName} from {xaml} not found in {theme.GetType()}");
+                Assert.NotNull(themeProperty);
+
+                Assert.Equal(solidColorBrush.Color, themeProperty.GetValue(theme));
+            }
+        }
+
+        [Fact]
+        public void VerifyLightThemeColorsInXaml()
+        {
+            AssertThemeColorsInXaml(new MaterialDesignLightTheme(), "MaterialDesignTheme.Light.xaml");
+        }
+
+        [Fact]
+        public void VerifyDarkThemeColorsInXaml()
+        {
+            AssertThemeColorsInXaml(new MaterialDesignDarkTheme(), "MaterialDesignTheme.Dark.xaml");
+        }
+
+        private static void AssertThemeColorsInXaml(IBaseTheme baseTheme, string xaml)
+        {
+            var theme = new Theme();
+            theme.SetBaseTheme(baseTheme);
+
+            var uri = new Uri("pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/" + xaml);
```

Comment:
There is an MdixHelper class that has some of this functionality already in it to load up the theme dictionaries. Might need to make some of the methods public to leverage them here.

Reply:
I have updated MdixHelper and the unit test.